cmd/antrea-agent/util.go

```diff
@@ -79,3 +88,54 @@ func parsePortRange(portRangeStr string) (start, end int, err error) {
 
 	return start, end, nil
 }
+
+func parseCIDRs(s string) []*net.IPNet {
+	s = strings.TrimSpace(s)
+	if s == "" {
+		return nil
+	}
+
+	var cidrs []*net.IPNet
+	for _, cidrStr := range strings.Split(s, ",") {
+		cidrStr = strings.TrimSpace(cidrStr)
+		if cidrStr == "" {
+			continue
+		}
+		_, cidr, err := net.ParseCIDR(cidrStr)
+		if err != nil {
+			klog.ErrorS(err, "Failed to parse CIDR", "cidrStr", cidrStr)
+			continue
+		}
+		cidrs = append(cidrs, cidr)
+	}
+	return cidrs
+}
+
+// getPodCIDRs gets the cluster-wide Pod CIDRs (IPv4 and IPv6) by attempting the following sources in order:
+// 1. Agent configuration.
+// 2. kube-proxy ConfigMap.
+// 3. kubeadm-config ConfigMap.
+func getPodCIDRStr(o *Options, k8sClient clientset.Interface) string {
+	// Get Pod CIDR from agent config
+	podCIDRStr := o.config.PodCIDR
```

Comment:
I suppose we should trim the space here? in case the podCIDR in the config is with space " " by accident instead of an empty value.